flu_virus.yaml

```diff
@@ -0,0 +1,46 @@
+# input deck for rendering one of the influenza A virion final
+# snapshots from https://doi.org/10.1016/j.str.2014.12.019
+# this system may be challenging for nice renders re: coloring
+# lipids and proteins differently, handling MARTINI/CG particles, etc.
+
+topology_path: "/Users/treddy/rough_work/ggmolvis/virion_FORS_323K_final_snapshot.pdb"
+trajectory_path: ""
+# normal options for blender_engine: "BLENDER_EEVEE_NEXT", "CYCLES"
+blender_engine: "CYCLES"
+# normal options for blender_engine_device: "CPU", "GPU"
+# only CYCLES can use CPU though
+blender_engine_device: "GPU"
+blender_resolution_x: 1000
+blender_resolution_y: 1000
+# mode can be "movie" or "image"
+mode: "image"
+# .mp4 or .png filename suffix will be appended automatically
+# depending on `mode`
+render_filename: "test_virion"
+frame_start: 0
+frame_end: 0
+sel_string: "protein"
+# NOTE: the RPO core isn't currently selected
+sel_string2: "resname PLM PPC FOR POP DOP CHO"
```

Comment:
This doesn't scale well and would require the user to adjust the Python source to add more selection string "receivers" for each selection added here. Also, the colors are currently set by hand in the Python source as well.

I'm not sure how heavily this YML format will actually be used though, so may not need to worry too much about exact design for the moment.

Reply:
Also important to note that these changes break the workflow for ADK of course, since it doesn't have `sel_string2`, so should try to make the YAML work with both targets..